stablehlo/tests/interpret_add.mlir

```diff
@@ -145,6 +145,21 @@ func.func @add_op_test_ui64() -> tensor<2xui64> {
 
 // -----
 
+// CHECK-LABEL: Evaluated results of function: add_op_test_i1
+func.func @add_op_test_i1() -> tensor<4xi1> {
+  %0 = stablehlo.constant dense<[false, false, true, true]> : tensor<4xi1>
+  %1 = stablehlo.constant dense<[false, true, false, true]> : tensor<4xi1>
+  %2 = stablehlo.add %0, %1 : tensor<4xi1>
+  func.return %2 : tensor<4xi1>
+  // CHECK-NEXT: tensor<4xi1>
+  // CHECK-NEXT: false
+  // CHECK-NEXT: true
+  // CHECK-NEXT: true
+  // CHECK-NEXT: false
```

Comment:
As discussed in #264, `true + true = true`. This is one example for why conflating `bool` and `APInt` seems risky.